diff --git a/node_modules/reka-ui/dist/shared/useFormControl.cjs b/node_modules/reka-ui/dist/shared/useFormControl.cjs
index 47d8e19..1ca8766 100644
--- a/node_modules/reka-ui/dist/shared/useFormControl.cjs
+++ b/node_modules/reka-ui/dist/shared/useFormControl.cjs
@@ -4,7 +4,7 @@ const core = require('@vueuse/core');
 const vue = require('vue');
 
 function useFormControl(el) {
-  return vue.computed(() => core.toValue(el) ? Boolean(core.unrefElement(el)?.closest("form")) : true);
+  return vue.computed(() => core.toValue(el) ? Boolean(core.unrefElement(el)?.closest("form[data-native]")) : true);
 }
 
 exports.useFormControl = useFormControl;
diff --git a/node_modules/reka-ui/dist/shared/useFormControl.js b/node_modules/reka-ui/dist/shared/useFormControl.js
index 2d71bfc..55c4b54 100644
--- a/node_modules/reka-ui/dist/shared/useFormControl.js
+++ b/node_modules/reka-ui/dist/shared/useFormControl.js
@@ -2,7 +2,7 @@ import { toValue, unrefElement } from '@vueuse/core';
 import { computed } from 'vue';
 
 function useFormControl(el) {
-  return computed(() => toValue(el) ? Boolean(unrefElement(el)?.closest("form")) : true);
+  return computed(() => toValue(el) ? Boolean(unrefElement(el)?.closest("form[data-native]")) : true);
 }
 
 export { useFormControl as u };
